DOCS-3002: Add instructions for physically connecting components to how-tos (#3535)
viamrobotics/docs: docs/how-tos/deploy-ml.md

{{% expand "A configured camera. Click to see instructions." %}}

> First, connect the camera to your machine's computer if it's not already connected (like with an inbuilt webcam on a Macbook).

Reviewer: (it's MacBook but we also don't have to be apple-specific)
```suggestion
First, connect the camera to your machine's computer if it's not already connected (like with an inbuilt laptop webcam).
```
